### ISSUE TITLE
Should service fee components be hidden from the front-end?

### ISSUE THREAD

**Issue by philipr-za** [big-neon/bn-web#1361]
While implementing the CC fee we were

**Comment by Krakaw** [comment]
I would weigh in that we should probably merge the service fees on the backend in the controller and send a single value to the client. We maintain the individual fees in the order_items table, but send a single value back for the client to render. @zmelamed what do you think?

**Comment by zmelamed** [comment]
That's correct, for the fees we currently display everything should be combined as a single Service Fee to the end user.
